gds-api-adapters.gemspec

```diff
@@ -21,24 +21,21 @@ Gem::Specification.new do |s|
   s.add_dependency "plek", ">= 1.9.0"
   s.add_dependency "rest-client", "~> 2.0"
 
-  s.add_development_dependency "climate_control", "~> 0.2"
+  s.add_development_dependency "climate_control", "~> 1.2"
   s.add_development_dependency "govuk-content-schema-test-helpers", "~> 1.6"
   s.add_development_dependency "minitest", "~> 5.11"
   s.add_development_dependency "minitest-around", "~> 0.5"
-  s.add_development_dependency "mocha", "~> 1.11"
-  s.add_development_dependency "pact", "~> 1.20"
-  s.add_development_dependency "pact_broker-client", "~> 1.14"
+  s.add_development_dependency "mocha", "~> 1.14"
+  s.add_development_dependency "pact", "~> 1.62"
+  s.add_development_dependency "pact_broker-client", "~> 1.65"
   s.add_development_dependency "pact-consumer-minitest", "~> 1.0"
-  s.add_development_dependency "pact-mock_service", "~> 3.6"
-  s.add_development_dependency "pry"
```

Comment:
I expect you have a solid rationale for removing this, so probably more out of interest for me - isn't there a case for leaving pry in despite it not being used, to aid development?

Reply:
I removed it because it's not referenced at all and there's no clear convention in gems for having a debugger or which one (some have byebug, some have pry-byebug etc). Not hugely opinionated on whether it should stay or go to be honest - sorry not much of a rationale 😄

Reply:
We wouldn't expect it to be referenced though would we? I.e. just used temporarily for development. Unless there's another step to enabling it for development beyond just this? If not, I'd maybe be tempted to keep it in as I suspect someone will re-add it at some point (although noted no consensus). But also not strongly opinionated either way.

Reply:
Yup, that's reasonable. I'll switch it to byebug, as that's at least the one we reference in https://docs.publishing.service.gov.uk/manual/setting-up-new-rails-app.html

Reply:
Added in https://github.com/alphagov/gds-api-adapters/pull/1159/commits/0a6ed791ba7b7d774588eb26af1c5b595c586805